app.js

```diff
@@ -14,12 +14,38 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(__dirname + '/views/partials');
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => res.render('beers', { beersFromApi }))
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => res.render('random-beer', { responseFromAPI }));
+});
+
+app.get('/beer-detailed', (req, res) => {
+  res.render('beer-detailed');
```

Comment:
This one isn't working because if you go to your beer-detailed page, you only have a #each beerApi and the partial you created. If you want to display that, you would need to use the punkAPI in here.
So, for example:

```
punkAPI
  .getBeers()
  .then(beerApi => res.render('beer-detailed' , {beerApi}))
  .catch(error => console.log(error))
```